src/libs/ajax/Metrics.ts

```diff
@@ -1,32 +1,25 @@
 import { getDefaultProperties } from '@databiosphere/bard-client';
 import _ from 'lodash/fp';
 import { authOpts, fetchBard, jsonBody } from 'src/libs/ajax/ajax-common';
-import { ensureAuthSettled } from 'src/libs/auth';
 import { getConfig } from 'src/libs/config';
 import { withErrorIgnoring } from 'src/libs/error';
 import { MetricsEventName } from 'src/libs/events';
 import * as Nav from 'src/libs/nav';
-import { authStore, getSessionId } from 'src/libs/state';
-import { v4 as uuid } from 'uuid';
+import { AuthState, authStore, getSessionId } from 'src/libs/state';
 
 export const Metrics = (signal?: AbortSignal) => {
-  const captureEventFn = async (event: MetricsEventName, details: Record<string, any> = {}): Promise<void> => {
-    await ensureAuthSettled();
```

Comment:
Why are we removing this?

Reply:
I am removing it because it seems like a smell to me because it modifies the state from within the metrics call. I think if you do care about waiting for the auth to be in a certain state before making the call, then you should call the metrics from that state (which is why I made the `AuthState` a optional param). Most metrics calls only happen once you are already logged in, so this would only affect the metrics calls that are made during that transition (which our the ones under our domain like login, register etc.)

Reply:
Then again... it might be fine to remove since the anonymous ID (initialized when the app loads) should get tied to the actual user ID when the user information is loaded.